tests/channel_test.c

```diff
@@ -239,15 +239,16 @@ static int s_test_channel_refcount(struct aws_allocator *allocator, void *ctx) {
     struct aws_channel_creation_callbacks callbacks = {
         .on_setup_completed = s_channel_setup_test_on_setup_completed,
         .setup_user_data = &test_args,
-        .on_shutdown_completed = s_channel_setup_test_on_setup_completed,
-        .shutdown_user_data = &test_args,
+        .on_shutdown_completed = NULL,
```

Comment:
Looks like a copy-past error. I assumed waiting on shutdown being complete isn't needed in this test so made them NULL.